config.ts

```diff
@@ -59,6 +59,7 @@ const ENV = {
     env: "dev",
     xmtpEnv: (process.env.EXPO_PUBLIC_DEV_XMTP_ENV || "dev") as XmtpEnv,
     apiURI: process.env.EXPO_PUBLIC_DEV_API_URI || "",
+    // apiURI: "https://fa7b225de0e1.ngrok.app/",
```

Comment:
When running from my physical device in dev environment, do we expect the device to be able to connect to the server running on the computer automatically?

I used this as a workaround to that not working.

Reply:
Only for iOS, iOS does some different stuff than android
Android allows port forwarding with adb
iOS uses I believe MDM, but I might be getting the abbreviation wrong. It requires the same Wifi and certain capabilities/lack of certain securities to work

Reply:
thansk for the context here alex. nice to have would be automatically tunneling on ios, but not going to do this now.